tekton/resources/nightly-tests/bastion-z/cleanup_tekton.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cleanup-tekton-nightly
+spec:
+  params:
+  - name: package
+  - name: resources
+    description: space separated list of resources to be deleted
+    default: "conditions pipelineresources tasks pipelines taskruns pipelineruns"
+  - name: kubeconfig-secret
+    description: secret with kubeconfig for remote cluster
+  resources:
+    inputs:
+    - name: plumbing-source
+      type: git
+    - name: tekton-project-source
+      type: git
+      targetPath: src/$(params.package)
```

Comment:
I'm slowly working through the process of removing pipeline resources in various places from our infra - since they're not really supported anymore. I guess this is fine, but eventually we should try and rely on workspaces and the git clone task instead. I don't like having two use a PVC and many pods for that, but that is something that we'll hopefully fix through changes in the runtime model.

Reply:
yep, moving to workspaces will be definitely todo for me too